.github/workflows/create-js-release.yaml

```diff
@@ -65,8 +65,8 @@ jobs:
             target: x86_64-unknown-linux-gnu
             architecture: x64
             docker: |
-              docker pull ghcr.io/napi-rs/napi-rs/nodejs-rust:lts-debian
-              docker tag ghcr.io/napi-rs/napi-rs/nodejs-rust:lts-debian builder
+              docker pull ghcr.io/napi-rs/napi-rs/nodejs-rust:sha256:e510d7fdb259bd9b7ce05aa961223dd6379c1a944f85c271b76561b7ab7565b2
```

Comment:
this uses the last good sha from 0.3.1. Idk what changed in the image, but the latest doesnt work anymore.

Reply:
@Brooooooklyn did anything change in `lts-debian` that would cause this to fail? My last release  was on feb 22. all of my other runs are successful. But the lts-debian has some issues with a copy error. 
current failure
https://github.com/pola-rs/polars/runs/5965443656?check_suite_focus=true#step:13:340

last success
https://github.com/pola-rs/polars/actions/runs/1885659523

Reply:
@universalmind303 Hey my bad, please upgrade to `@napi-rs/cli@2.6.2` and `nodejs-rust:lts-debian`. In image it fix a linker error which caused `cannot allocate memory in static TLS block` in `worker_threads`. But it introduced an error in `@napi-rs/cli` but fixed in the latest version.

Reply:
I'm finding a better way to test the Docker images.